ws-nextjs-app/jest.config.mjs

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/extensions */
+import { createRequire } from 'module';
+import nextJest from 'next/jest.js';
+import pkg from 'ts-jest/utils/index.js';
+
+const { pathsToModuleNameMapper } = pkg;
+
+const require = createRequire(import.meta.url);
+const { compilerOptions } = require('../tsconfig.json');
```

Comment:
One options is perhaps to convert the tsconfig.json to a typescript file instead?

Reply:
Yea its a bit of workaround to import the JSON file in an ES6 file. Could convert it to a CommonJS file, but would be a small step backwards to ES5 I guess? Not a huge deal since its a config file.

The `tsconfig.json` will need to remain as such as its used by the TypeScript complier and IDE.

Reply:
Can we use this in our TS config? https://www.typescriptlang.org/tsconfig#resolveJsonModule

Reply:
Or what about this - https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#tsconfig-bases?

Reply:
We have `resolveJsonModule` set in the root `tsconfig.json` which the Next app inherits from but doesn't seem to resolve the issue here.

The import syntax here _may_ get resolved once we move to Node 18 as the `assert` syntax is supported there. So we could do:
```
import myJson from './example.json' assert {type: 'json'}
```